Api-Layer-Java.ppf

```diff
@@ -0,0 +1,50 @@
er-ui/2.8.0/2b895560df9cd70a2e93119188981b004b4e2ad5/springfox-swagger-ui-2.8.0.jar:../../../.gradle/caches/modules-2/files-2.1/io.springfox/springfox-swagger2/2.8.0/27fbca30912258ce0e31cd56361f732f6c27d5ff/springfox-swagger2-2.8.0.jar:../../../.gradle/caches/modules-2/files-2.1/io.springfox/springfox-schema/2.8.0/70ed8275e2277bffe269c8dbd688f54464bf1222/springfox-schema-2.8.0.jar:../../../.gradle/caches/modules-2/files-2.1/io.springfox/springfox-spi/2.8.0/efe49b45357dd9f6c27799a5f7f5ba1ce6bdc05f/springfox-spi-2.8.0.jar:../../../.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/2.23.4/a35b6f8ffcfa786771eac7d7d903429e790fdf3f/mockito-core-2.23.4.jar:../../../.gradle/caches/modules-2/files-2.1/org.zowe.explorer.api/explorer-api-common-test/0.3.1-SNAPSHOT/b7ae1f7864e0cb9c29734773454a651d3415790a/explorer-api-common-test-0.3.1-SNAPSHOT.jar:../../../.gradle/caches/modules-2/files-2.1/org.zowe.explorer.api/explorer-api-common-test/0.3.2-SNAPSHOT/442ce506cc7a717b495b0a454b9e184ed36be47b/explorer-api-common-test-0.3.2-SNAPSHOT.jar:../../../.gradle/caches/modules-2/files-2.1/org.zowe.explorer.api/explorer-api-common/0.3.1-SNAPSHOT/bb050e0e7b3a971cfdcad5617f9c4373d6c5635b/explorer-api-common-0.3.1-SNAPSHOT.jar:../../../.gradle/caches/modules-2/files-2.1/org.zowe.explorer.api/explorer-api-common/0.3.2-SNAPSHOT/a3251812d143df4247cf86be63472e45786fa05c/explorer-api-common-0.3.2-SNAPSHOT.jar:../../../.gradle/caches/modules-2/files-2.1/org.json/json/20140107/d1ffca6e2482b002702c6a576166fd685e3370e3/json-20140107.jar:../../../.gradle/caches/modules-2/files-2.1/commons-logging/commons-logging/1.2/4bfc12adfe4842bf07b657f0369c4cb522955686/commons-logging-1.2.jar:../../../.gradle/caches/modules-2/files-2.1/commons-logging/commons-logging/1.1.1/5043bfebc3db072ed80fbd362e7caf00e885d8ae/commons-logging-1.1.1.jar:../../../.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-analyzers-common/5.5.5/e6b3f5d1b33ed24da7eef0a72f8062bd4652700c/lucene-analyzers-common-5.5.5.jar:../../../.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-queryparser/5.5.5/6c965eb5838a2ba58b0de0fd860a420dcda11937/lucene-queryparser-5.5.5.jar:../../../.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-queries/5.5.5/d99719e7c58c149113f897bca301f1d68cbf3241/lucene-queries-5.5.5.jar:../../../.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-sandbox/5.5.5/d145d959109257c47151be43b211213dff455f47/lucene-sandbox-5.5.5.jar:../../../.gradle/caches/modules-2/files-2.1/org.apache.lucene/lucene-core/5.5.5/c34bcd9274859dc07cfed2a935aaca90c4f4b861/lucene-core-5.5.5.jar:../../../.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.9.6/456895fc91bf7180b216fead220373e6278230c9/jackson-datatype-jdk8-2.9.6.jar:../../../.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.9.6/ea54f6193d224e5e5732bbd4262327eb465397c2/jackson-datatype-jsr310-2.9.6.jar" jdk_name="IBM JDK 1.8" name="Configuration 1"/>
```

Comment:
May be rename configuration name to more understandable, like `Appscan Configuration`, or something similar...

Reply:
I only updated the files containing our project modules that need to be tested, I wouldn't change anything else in the file cause we cannot know the impact. 
These files are needed for the Appscan automation by the CI/CD squad. I don't think we should change naming cause we cannot see the rest of their automation, and they might expect this configuration name.

Reply:
Yes, I understand that, but we can ask them about that, right?

Reply:
@JirkaAichler wanted to ask some additional stuff too, maybe sent them an email, or address them on previous pull request.